5g-control-plane/Chart.yaml

```diff
@@ -10,7 +10,7 @@ description: SD-Core 5G control plane services
 name: 5g-control-plane
 icon: https://guide.opencord.org/logos/cord.svg
 
-version: 1.0.3
+version: 1.0.4
```

Comment:
May be we can update helm chart after SMF, AMF image release?

Reply:
Agree